### ISSUE TITLE
False positive on SpaceInParentheses inside strings

### ISSUE THREAD

**Issue by lasseebert** [rrrene/credo#522]
### Environment

* Credo version (`mix credo -v`): 0.9.0
* Erlang/Elixir version (`elixir -v`): 1.6.3
* Operating system: Ubuntu

### What were you trying to do?

```
def foo do
  " )"
end
```

### Expected outcome

No credo warnings

### Actual outcome

```
[C] ↗ There is no whitespace around parentheses/brackets most of the time, but 
      here there is.
```

**Comment by rrrene** [comment]
@lasseebert Thx for reporting! Unfortunately, I can't reproduce this bug.

When I just add

```elixir
def foo do
  " )"
end
```

to [the relevant consistency tests](https://github.com/rrrene/credo/blob/6aeb7bb7d8ec52bf798125e85dad194f3ddef686/test/credo/check/consistency/space_in_parentheses_test.exs#L7) nothing changes.

In the code where this happens, might there be sigils with modifiers in the same file? Like `~r/foo/iu`?

**Comment by lasseebert** [comment]
There are no sigils in the same file. Will try to produce a small example that reproduces the issue in a brand new mix project.

**Comment by rrrene** [comment]
The fix for this was pretty spectacular 😀  It is now fixed on `master`.

You can try this by setting the Credo dep to

```elixir
{:credo, github: "rrrene/credo"}
```

Please report back if your issue is solved! 👍
